.circleci/config.yml

```diff
@@ -652,7 +652,7 @@ jobs:
             pip install --user codespell
       - run:
           name: Check spelling
-          command: ~/.local/bin/codespell -S "*.enc,.git,Dockerfile*" -I ./scripts/codespell_whitelist.txt
+          command: ~/.local/bin/codespell -S "*.enc,.git,Dockerfile*,LICENSE" -I ./scripts/codespell_whitelist.txt
```

Comment:
Maybe we should ignore the whitelist as well? :) Messages on the channel indicate that it's being spellchecked as well.